src/manage_nsfs/manage_nsfs_cli_responses.js

```diff
@@ -31,7 +31,11 @@ class ManageCLIResponse {
                 code: this.code,
             }
         };
-        if (this.list || this.status) json.response.reply = typeof detail === 'string' ? JSON.parse(detail) : detail;
+        if (this.list || this.status) {
+            json.response.reply = typeof detail === 'string' ? JSON.parse(detail) : detail;
+        } else {
+            json.response.message = typeof detail === 'string' ? detail : `${detail}`;
```

Comment:
what is the difference between reply and message?

Reply:
reply is of JSON type and message should be one liner string explaination for the operation

Reply:
Maybe you can add this as a comment.

Reply:
Sure.